lib/mock-module/mock-module.spec.ts

```diff
@@ -44,3 +58,60 @@ describe('MockModule', () => {
     expect(mockedComponent).not.toBeNull();
   });
 });
+
+describe('SameImportsModules', () => {
+  let fixture: ComponentFixture<SameImportsComponent>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [
+        SameImportsComponent
+      ],
+      imports: [
+        MockModule(SameImports1Module),
+        MockModule(SameImports2Module),
+      ],
+    })
+    .compileComponents()
+    .then(() => {
+      fixture = TestBed.createComponent(SameImportsComponent);
+      fixture.detectChanges();
+    });
+  }));
+
+  it('should be imported correctly', () => {
+    expect(fixture.componentInstance).toEqual(jasmine.any(SameImportsComponent));
+    expect(fixture.nativeElement.innerText).toEqual('same imports');
+  });
+});
+
+// TODO> Doesn't work because ParentModule doesn't export anything.
+// TODO> Basically it's feature of ng-mocks to export declarations of mocked modules.
+// describe('RealModule', () => {
```

Comment:
I would suggest to keep this test so people can understand the difference between mocked module and real one.